k8s/contiv-vpp/README.md

```diff
@@ -0,0 +1,32 @@
+# Contiv VPP
+
+Deploy Contiv VPP networking
+
+The deployment consists of the following components:
+ * contiv-etcd - deployed on the master node
+ * contiv-vswitch - deployed on each node
+ * contiv-ksr - deployed on the master node
+
+The `values.yaml` file contains the default values for the
+contiv-vpp templates.
+
+## Installing the chart
+
+To install the chart with the release name `my-release`:
+
+```console
+$ helm install --name my-release contiv-vpp
+```
+## Uninstalling the chart
+
+```console
+$ helm delete --purge my-release
+```
+
+
+## Configuration
+
+The following tables lists the configurable parameters of the Contiv-VPP chart and their default values.
+
+Parameter | Description | Default
+--------- | ----------- | -------
```

Comment:
missing!  esp for how to set nodeconfig
maybe can point to the values.yml and indicate how to override on helm install?

Reply:
I documented all of the options in values.yaml and put a pointer to commented section for the node config stuff. Many of those options can use a better description but it is a good placeholder for now.

Reply:
thx

Reply:
Can we also checkin the default generated manifest, so that folks who do not want to install tiller do not have to install it? we must provide a simple, "easy-button" installation for folks who just want to play with it or are doing it for the first time.